apache.dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM php:8.0.1-apache
+
+
```

Comment:
This docker file doesn't look like it's copying the LORIS code into the docker image.

Reply:
This docker file builds a PHP-apache environment to set up Loris.
Loris code bind to docker with volume.
   " volumes:
      - .:/var/www/html
"

Reply:
This isn't resolved. We need a docker file that is a docker image of LORIS. It needs to include LORIS and be self-contained.